spec/services/invoices/payments/adyen_service_spec.rb

```diff
@@ -0,0 +1,244 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Invoices::Payments::AdyenService, type: :service do
+  subject(:adyen_service) { described_class.new(invoice) }
+
+  let(:customer) { create(:customer) }
+  let(:organization) { customer.organization }
+  let(:adyen_payment_provider) { create(:adyen_provider, organization:) }
+  let(:adyen_customer) { create(:adyen_customer, customer:) }
+  let(:adyen_client) { instance_double(Adyen::Client) }
+  let(:payments_api) { Adyen::PaymentsApi.new(adyen_client, 70) }
+  let(:checkout) { Adyen::Checkout.new(adyen_client, 70) }
+  let(:payments_response) { generate(:adyen_payments_response) }
+  let(:payment_methods_response) { generate(:adyen_payment_methods_response) }
+
+  let(:invoice) do
+    create(
+      :invoice,
+      organization:,
+      customer:,
+      total_amount_cents: 1000,
+      currency: 'USD',
+      ready_for_payment_processing: true
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)